.ci/manywheel/build_cuda.sh

```diff
@@ -56,13 +56,13 @@ EXTRA_CAFFE2_CMAKE_FLAGS+=("-DATEN_NO_TEST=ON")
 case ${CUDA_VERSION} in
     #removing sm_50-sm_70 as these architectures are deprecated in CUDA 12.8/9 and will be removed in future releases
     12.8)
-        TORCH_CUDA_ARCH_LIST="7.5;8.0;8.6;9.0;10.0;12.0"
+        TORCH_CUDA_ARCH_LIST="6.0;7.0;7.5;8.0;8.6;9.0;10.0;12.0"
         ;;
     12.9)
-        TORCH_CUDA_ARCH_LIST="7.5;8.0;8.6;9.0;10.0;12.0+PTX"
+        TORCH_CUDA_ARCH_LIST="6.0;7.0;7.5;8.0;8.6;9.0;10.0;12.0+PTX"
```

Comment:
Adding 6.0 and 7.0 is not an option according to: https://github.com/pytorch/pytorch/issues/157517 and https://github.com/pytorch/pytorch/issues/157517#issuecomment-3046062047. Please remove it

Reply:
It wasn't until we reduced the size with the compression flags. Now everything seems like it might work (we at least aren't hitting the libtorch limit